components/ListView/ListItemComponents.js

```diff
@@ -10,22 +10,9 @@ class ListItemComponents extends React.Component {
     this.state = { expanded: false };
   }
 
-  componentWillReceiveProps(newProps) {
-    // compare old value to new value, and if this component is getting new data,
-    // then get the current expand state of the new value as it is in the old dom
-    // and apply that expand state to this component
-    const olditem = this.props.listItem.name;
-    const newitem = newProps.listItem.name;
-    const parent = this.props.listItemParent;
-    if (olditem !== newitem) {
-      if ($(`.${parent} [data-component="${newitem.name}"]`).hasClass("active")) {
-        this.setState({ expanded: true });
-      } else {
-        this.setState({ expanded: false });
-      }
-    }
```

Comment:
@jgiardino I believe when the props update (without a remount) the state won't be cleared. So, if the component has a new name and was already set as expanded it will remain expanded without this. Please let me know if I misconstrued the purpose of this method.

Reply:
This isn't needed. I was looking at this yesterday, and also ended up pulling this out as part of #802.